geometry.go

```diff
@@ -26,6 +26,11 @@ func (s1 Size) Union(s2 Size) Size {
 	return NewSize(maxW, maxH)
 }
 
+// FitsInto returns whether the current Size fits into another Size or not.
+func (s1 Size) FitsInto(s2 Size) bool {
```

Comment:
It has to be `SmallerOrEqualThan` then. It's quite long but if you prefer this, I'll change it.

Reply:
hmm, not sure I understood that it could be equal to from "FitsInto" either - after all a box the same size as another will not fit into it ;).
Perhaps there is something in between? Short but more easily guessable...

Reply:
Honestly, I am not sure of `FitsInto` at all. I was just the question that the caller wanted to be answered.

Reply:
If you're unsure this could be removed - as it's only called once it's probably not worth the trouble :)

Reply:
I've done so.